css-ruby-1/Overview.bs

```diff
@@ -647,23 +647,25 @@ Ruby Layout</h2>
 	its [=base level=] is laid out on the line,
 	aligned according to its 'vertical-align' property
 	exactly as if its [=ruby bases=] were a regular sequence of inline boxes.
-	Each [=ruby base container=] is sized and positioned
-	to contain exactly all of its [=ruby bases=]’ margin boxes.
+	The [=content box=] of each [=ruby base container=] is sized and positioned
+	to contain exactly all of its [=ruby bases=]’ [=margin boxes=].
```

Comment:
Is "contain exactly" intended to mean that it is the _smallest_ size possible that completely contains the margin boxes? It might be worth being explicit about this, since any extension relative to that size would still contain the boxes. 

It is probably implicit that the content box is a simple rectangle, but given that the union of margin boxes might not be a simple rectangle, consider making it explicit.

Same comments for ruby annotation container's content box size, line 659.

Reply:
I raised the request for a defined term "contains exactly" at #4950.